modules/ddl-and-loading/pages/creating-a-loading-job.adoc

```diff
@@ -70,6 +72,15 @@ CREATE LOADING JOB job_name FOR GRAPH Graph_Name {
 <1> A loading job may contain either `LOAD` or `DELETE` statements but not both.
 A loading job that includes both will be rejected when the `CREATE` statement is executed.
 
+=== Loading data to global vertices and edges
+
+The `CREATE` command creates global vertices and edges.
+Global vertex and edge type data is shared across all graphs, no matter which graph it is loaded to.
+In other words, if `v1` and `e2` are of `GLOBAL` type, the data loaded in this loading job will be accessible in other graphs that use `v1` and `e2` vertices and edges.
+
+To create a `LOCAL` vertex or edge, run a schema change job after a graph has already been created and use the `ADD` command to add a vertex or edge to a single graph.
```

Comment:
I don't think "local" is a GSQL word? We shouldn't need to put it in backticks. 

And don't say `ADD` command. Always complete the commands you are describing. `ADD VERTEX` and `ADD EDGE`